pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -219,6 +224,7 @@ def test_agg_multiple_functions_too_many_lambdas(df):
         grouped.agg(funcs)
 
 
+@pytest.mark.filterwarnings("ignore:NDFrame:FutureWarning")
```

Comment:
did you add all of these? we certainly don't want to do this

Reply:
I deprecated NDFrame.to_dense in generic.py, which caused warnings all throughout testing. Is there an alternative?

Reply:
@jreback Would it be fine if I filter warnings for only that method instead of all FutureWarnings, as such: `@pytest.mark.filterwarnings("ignore:NDFrame.to_dense:FutureWarning")`

Reply:
Yes, the problem is that the current filter is too general, and it should filter only the warnings you are adding in this PR.

Reply:
@jorisvandenbossche gotcha, will update accordingly soon.